src/ripple/net/impl/RPCCall.cpp

```diff
@@ -202,6 +218,25 @@ class RPCParser
         return v;
     }
 
+    Json::Value parseManifest (Json::Value const& jvParams)
+    {
+        if (1 == jvParams.size ())
+        {
+          Json::Value     jvRequest (Json::objectValue);
```

Comment:
Two style nits:
1) should indent four spaces
2) should be only one space after the type